pkg/ansible/run.go

```diff
@@ -64,23 +65,34 @@ func printVersion() {
 func Run(flags *aoflags.AnsibleOperatorFlags) error {
 	printVersion()
 
+	cfg, err := config.GetConfig()
+	if err != nil {
+		log.Error(err, "Failed to get config.")
+		return err
+	}
+
 	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
```

Comment:
This code block is rather ugly and should be moved below `manager.Options` construction so you can handle namespace logic all in one place. Ditto for helm.

Reply:
Agreed. 👍